### PR TITLE
LFR-1670: Add district card

### PR DESCRIPTION
**Pull request by zlahham**
Adds **District Card**

### CODE REVIEW

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.district {
+    width: 100%;
+    height: 100%;
+    border: solid 1px #e7e7e7;
+    background-color: #fdfdfd;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Check for closest color from Bloom

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.info {
+    margin: 1rem;
+    padding: 0;
+}
+
+.title {
+    font-family: var(--font-avenir);
+    font-size: var(--fontsize-large-i);
+    font-weight: 600;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Check for closest color from Bloom

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    font-stretch: normal;
+    font-style: normal;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Remove these two

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    line-height: 1.47;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Check if this is needed

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    letter-spacing: normal;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Check if this is needed

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    color: var(--color-black);
+    margin: 0 0 0.25rem 0;
+    padding: 0;
+    text-decoration: none;
+}
+
+.subtitle {
+    object-fit: contain;
```
Comment (zlahham) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: Not needed

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css (hunk 2 above)
Comment (jamero102) [reply packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: var(--fontweight-demi)

Comment (zlahham) [reply packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: demi is 700 :(

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 7]
```diff
@@ -0,0 +1,38 @@
+.district {
+  width: 100%;
+  height: 100%;
+  border: solid 1px #e7e7e7;
```
Comment (lukeamarsh) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: same for this with a bloom variable please :)

File: packages/core/src/components/Cards/DistrictCard/DistrictCard.css [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+  color: var(--color-black);
+  margin: 0 0 0.25rem 0;
+  padding: 0;
+  text-decoration: none;
+}
+
+.subtitle {
+  font-family: var(--font-avenir);
+  font-size: var(--font-small-i);
+  line-height: 1.43;
```
Comment (lukeamarsh) [inline packages/core/src/components/Cards/DistrictCard/DistrictCard.css]: are these line heights definitely needed? Might be best to use rems if possible

**Pull request merged**